dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -269,67 +269,42 @@
                             <span class="glyphicon glyphicon-download-alt"/> #{bundle.download} <span class="caret"/>
                         </button>
                         <ul jsf:id="downloadDropdownOptions" class="dropdown-menu multi-level pull-right text-left" role="menu">
-                            <li >
-                                <p:commandLink rendered="#{!(datasetFilesTab.downloadPopupRequired)}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                update="@form"
-                                                actionListener="#{datasetFilesTab.validateFilesForDownload(false, true)}">
-                                    #{bundle.downloadOriginal}
-                                </p:commandLink>
-                                <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                                <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                action="#{datasetFilesTab.validateFilesForDownload(true, true)}"
+                            <li>
+                                <p:commandLink disabled="#{false and datasetFilesTab.lockedFromDownload}"
+                                                action="#{datasetFilesTab.validateFilesForDownload(true)}"
                                                 update="@form">
                                     #{bundle.downloadOriginal}
                                 </p:commandLink>
                             </li>
-                            <li> <!--jsf:rendered="# {datasetFilesTab.isTabularDataSelected()}"-->
-                                <p:commandLink rendered="#{!(datasetFilesTab.downloadPopupRequired)}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                update="@form"
-                                                actionListener="#{datasetFilesTab.validateFilesForDownload(false, false)}">
-                                    #{bundle.downloadArchival}
-                                </p:commandLink>
-                                <!-- guest book or terms of use, etc. enabled - open "download popup" first: -->
-                                <p:commandLink rendered="#{datasetFilesTab.downloadPopupRequired}"
-                                                disabled="#{false and datasetFilesTab.lockedFromDownload}"
-                                                action="#{datasetFilesTab.validateFilesForDownload(true, false)}"
+                            <li>
+                                <p:commandLink disabled="#{false and datasetFilesTab.lockedFromDownload}"
```

Comment:
If always false then why bother with second statement?

Reply:
Ok. I will remove it.